src/main/ipc/docker.ts

```diff
@@ -195,6 +196,40 @@ async function execAmman() {
   log('Exiting execAmman');
   return p;
 }
+type ImageTagsResult = {
+  count: number;
+  results: HubImageTag[];
+};
+type HubImageTag = {
+  name: string;
+};
+
+async function getImageTags(imageName: string): Promise<string[]> {
+  const url = `https://hub.docker.com/v2/repositories/${imageName}/tags/`;
+
+  const promise = new Promise<string>((resolve, reject) => {
```

Comment:
This is OK for now, I think if we do more we should maybe look at [axios](https://www.npmjs.com/package/axios) for a promise based wrapper. I _think_ that's what the cool kids are doing

Reply:
Ah yea also, a good thing to know about, is Node's [util.promisify](https://nodejs.org/api/util.html). Won't work for `https.get`, but for other items like the Node exec library, you can wrap those with a promise using that helper.